### ISSUE TITLE
a11y: "main landing section divider" image has alt text

### ISSUE THREAD

**Issue by Danielle254** [kepsteen/great-harvest-landing-page#38]
Decorative images like this one should have empty alt text so that they are not picked up by screen readers. Also, this is still being rendered on large screens through use of `hidden` style attribute. It should not be rendered at all `display: none`.

![Image](https://github.com/user-attachments/assets/071fee03-c26a-4a89-98ce-abf5a7eb678b)
